Taskfile.yml

```diff
@@ -182,20 +183,24 @@ tasks:
     cmds:
```

Comment:
@haiqi96 I noticed we have this `platform` attribute for task `webui`. Does it mean the task only runs on x86 platforms? Now that we also set up the arch-specific meteor and nodejs, do we still need the restriction here? (My insights are limited as I'm not too sure how the webui is built currently for your workflow

Reply:
Does `meteor build --architecture` now support ARM platform IDs?

Reply:
Sorry for the oversight. It doesn't seem to be the case. 
The aarch64 related handling was added into the [install script](https://install.meteor.com/) together with the [2.14](https://web.archive.org/web/20240206033327/https://install.meteor.com/) -> 2.15 upgrade so I mistakenly thought it was available for download. https://static.meteor.com/packages-bootstrap/2.15/meteor-bootstrap-os.linux.aarch64.tar.gz just doesn't open. 

ARM support is expected to be available in [meteor 3.0](https://github.com/meteor/meteor/blob/3e6a51614a93d288d54d21cbcabe2c55b41332ef/meteor#L37) though.